src/network-interface.js

```diff
@@ -4,7 +4,10 @@ import {extractRequestFiles} from './helpers'
 export class HTTPUploadNetworkInterface extends HTTPFetchNetworkInterface {
   fetchFromRemoteEndpoint ({request, options}) {
     // Skip upload proccess if SSR
-    if (typeof window !== 'undefined') {
+    if (
+      (typeof window !== 'undefined') ||
+      (typeof navigator != 'undefined' && navigator.product == 'ReactNative')
```

Comment:
Just realised that the checks appear in a few files, and in this one are not repeated.